projects/ui/src/Components/Home/HomePage.tsx

```diff
@@ -27,8 +28,11 @@ export function HomePage() {
           description={`Welcome to the ${companyName} Developer Portal. Connect, partner, and build with us to create the next generation of digital experiences.`}
           additionalContent={
             !isAdmin && (
-              <NavLink to="/apis">
-                <Button style={{ width: "150px", marginTop: "10px" }}>
+              <NavLink to="/apis" onClick={onApisPageClick}>
+                <Button
+                  tabIndex={0}
```

Comment:
`tabIndex=0` causes the button to not get focused by keyboard input (when tabbing through the page).

I added this since the link gets focused, and this is nested, so if it's not there it's a strange keyboard navigation experience.